testing/e2e/app/routes/TransferList/spec.ts

```diff
@@ -0,0 +1,23 @@
+import { test, expect } from '@playwright/test';
+
+test('should support toolbar keyboard navigation in TransferList.Toolbar', async ({
+  page,
+}) => {
+  await page.goto('/TransferList');
+  const buttons = page.locator('button');
+
+  await page.keyboard.press('Tab');
+
+  await page.keyboard.press('Tab');
```

Comment:
After pressing <kbd>Tab</kbd>, we should validate focus. This will help with readability (knowing what we are tabbing to) and with maintainability (if we add more tab stops, this test will be easier to modify).

Also, it would be good to validate `role="toolbar"` somewhere. It feels brittle that we are looking for _all_ buttons on the page.